IdentityCore/src/MSIDOAuth2Constants.h

```diff
@@ -107,6 +107,7 @@ extern NSString *const MSID_CLIENT_INFO_CACHE_KEY;
 extern NSString *const MSID_ID_TOKEN_CACHE_KEY;
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->